internal/core/string.rs

```diff
@@ -288,6 +288,24 @@ pub fn format(args: core::fmt::Arguments<'_>) -> SharedString {
     output
 }
 
+/// A trait for converting a value to a [`SharedString`].
+///
+/// This trait is automatically implemented for any type which implements the [`Display`](https://doc.rust-lang.org/std/fmt/trait.Display.html) trait as long as the trait is in scope.
+/// As such, `ToSharedString` shouldn’t be implemented directly: [`Display`](https://doc.rust-lang.org/std/fmt/trait.Display.html)should be implemented instead, and you get the `ToSharedString` implementation for free.
```

Comment:
You can just write \`[Display]\`, there's no need to use a hard-coded absolute link. rustdoc creates the link. At least it did for me. Does it for you, too?